analytics/main_layout.js

```diff
@@ -16,14 +16,9 @@ setTimeout(function () {
   analytics.track('time on page more than 3 minutes', { category: '3 Minutes', message: sd.CURRENT_PATH })
 }, 180000)
 
-// debug tracking calls in development
+// debug tracking calls in development and staging
 if (sd.NODE_ENV !== 'production') {
   analytics.on('track', function () {
     console.debug('TRACKED: ', arguments[0], JSON.stringify(arguments[1]))
   })
 }
-if (sd.NODE_ENV === 'development') {
-  analyticsHooks.on('all', function (name, data) {
-    console.info('ANALYTICS HOOK: ', name, data)
```

Comment:
So the reason we use `debug` and `log` is that you can filter these out for clarity.

e.g.

![image](https://cloud.githubusercontent.com/assets/555859/21437561/7fc14f44-c852-11e6-8365-1e85307b68ae.png)